Taskfile.yaml

```diff
@@ -70,6 +70,11 @@ tasks:
     cmds:
       - docker compose exec postgres psql -U postgres -d ferretdb
 
+  mongo:
+    desc: "Run legacy mongo shell"
```

Comment:
Should it be the same as in https://github.com/FerretDB/FerretDB/pull/1600/files#diff-cd2d359855d0301ce190f1ec3b4c572ea690c83747f6df61c9340720e3d2425e?